backend/coreapp/views/scratch.py

```diff
@@ -305,5 +305,16 @@ def export(self, request: Request, pk):
             }
         )
 
+    @action(detail=True)
+    def family(self, request: Request, pk: str) -> Response:
+        scratch: Scratch = self.get_object()
+
+        family = Scratch.objects.filter(
+            target_assembly=scratch.target_assembly,
+            compiler=scratch.compiler,
+        )
+
+        return Response(ScratchSerializer(family, many=True, context={ 'request': request }).data)
```

Comment:
can't this exclude some fields / use the terse scratch serializer? being able to grab an entire family including all fields in one go seems like a really bulky request

Reply:
Whoops, I meant to use terse here